.github/workflows/publish-draft-releases.yml

```diff
@@ -340,11 +340,405 @@ jobs:
       #   run: |
       #     jq --slurp '. | add' ${{ github.workspace }}/extrinsic-benchmark-pallet-manta-pay-*.json > ${{ github.workspace }}/extrinsic-benchmark-pallet-manta-pay.json
 
+  build-node-old:
+    needs:
+      - start-node-builder-old
+      - check-for-runtime-upgrade
+    runs-on: ${{ needs.start-node-builder-old.outputs.runner-label }}
+    env:
+      CARGO_TERM_COLOR: always
+    steps:
+      -
+        if: ${{ needs.check-for-runtime-upgrade.outputs.runtime-version != needs.check-for-runtime-upgrade.outputs.runtime-version-old }}
+        uses: actions/checkout@v2
+        with:
+          ref: '4204135e90ce02fa28af0913c8f050b76bd3ea23'
```

Comment:
This should be ``manta-pc``.
Using this commit for testing as it has my modified chain spec.

TODO: revert before merging.